libs/barista-components/sunburst-chart/src/sunburst-chart.ts

```diff
@@ -39,61 +36,96 @@ import {
   DtUiTestConfiguration,
   DT_UI_TEST_CONFIG,
 } from '@dynatrace/barista-components/core';
-import { DtSunburstOverlay } from './sunburst.directive';
+import { DtSunburstChartOverlay } from './sunburst-chart.directive';
 import {
-  DtSunburstNode,
-  DtSunburstNodeInternal,
-  DtSunburstSlice,
-  DtSunburstValueMode,
+  DtSunburstChartTooltipData,
+  DtSunburstChartNode,
+  DtSunburstChartSlice,
   fillNodes,
   getNodesWithState,
   getSelectedId,
   getSelectedNodes,
   getSelectedNodesFromOutside,
   getSlices,
   getValue,
-} from './sunburst.util';
+} from './sunburst-chart.util';
+import { Platform } from '@angular/cdk/platform';
 
-const OVERLAY_PANEL_CLASS = 'dt-sunburst-overlay-panel';
+const OVERLAY_PANEL_CLASS = 'dt-sunburst-chart-overlay-panel';
 
+/**
+ * Sunburst chart is a donut chart with multiple levels that get unfolded on click and show an overlay on hover
+ */
 @Component({
-  selector: 'dt-sunburst',
-  templateUrl: 'sunburst.html',
-  styleUrls: ['sunburst.scss'],
+  selector: 'dt-sunburst-chart',
+  templateUrl: 'sunburst-chart.html',
+  styleUrls: ['sunburst-chart.scss'],
   preserveWhitespaces: false,
   changeDetection: ChangeDetectionStrategy.OnPush,
   encapsulation: ViewEncapsulation.Emulated,
+  host: {
+    '(click)': '_onClick($event)',
+  },
 })
-export class DtSunburst implements OnChanges {
-  readonly width = 480;
-  readonly viewBox = '-240 -176 480 352';
-
-  @ViewChild('svg') svgEl;
-
-  /** @internal Template reference for the DtSunburst overlay. */
-  @ContentChild(DtSunburstOverlay, { read: TemplateRef })
-  // tslint:disable-next-line: no-any
-  private _overlay: TemplateRef<any>;
-
-  filledSeries: DtSunburstNodeInternal[];
-  slices: DtSunburstSlice[];
-
-  _selected: DtSunburstNodeInternal[];
-  selectedLabel: string;
-  selectedValue: number;
-  selectedRelativeValue: number;
-  labelAsAbsolute: boolean = true;
+export class DtSunburstChart {
+  private _viewBoxWidth = 480;
+  private _viewBoxHeight = 400;
```

Comment:
those two don't have to be members I guess - you can create constants outside the class :)